packages/lmi/src/lmi/llms.py

```diff
@@ -833,6 +882,17 @@ async def acompletion(self, messages: list[Message], **kwargs) -> list[LLMResult
         used_model = completions.model or self.name
         results: list[LLMResult] = []
 
+        usage = getattr(completions, "usage", None)
+        prompt_count = usage.prompt_tokens if usage else 0
```

Comment:
Wait can we remove this `else 0`? Why do we have it?

If we need it, can you add a code comment stating why we have it

Let's not maintain backwards compatibility here, if that's the reason